cov.el

```diff
@@ -89,6 +90,16 @@ See `fringe-bitmaps' for a full list of options"
   :group 'cov
   :type 'symbol)
 
+(defcustom cov-highlight-lines nil
+  "Hightlight lines."
```

Comment:
Considering this is a customization option I think it should have much better documentation.

Reply:
That's right!, I was a bit too exited to test the new overlays!